open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/graph-repository-connector/pom.xml

```diff
@@ -23,6 +23,18 @@
     <artifactId>graph-repository-connector</artifactId>
 
     <dependencies>
+
+        <dependency>
+            <groupId>org.janusgraph</groupId>
+            <artifactId>janusgraph-cassandra</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>hibernate-validator</artifactId>
```

Comment:
Same comment here on exclusions

Reply:
Reverted, no longer applies

Reply:
This version of the validator will overwrite another and the result is Egeria not starting, failing with NoSuchMethod excepion, or similar

Reply:
Reproduced:
```
Caused by: org.springframework.beans.BeanInstantiationException: Failed to insta                                     ntiate [org.springframework.validation.beanvalidation.LocalValidatorFactoryBean]                                     : Factory method 'defaultValidator' threw exception; nested exception is java.la                                     ng.NoClassDefFoundError: org.hibernate.validator.internal.engine.ConfigurationIm                                     pl (initialization failure)
```

Reply:
Looks like later versions of the validator cause spring issues. I'd concur that for now we go with the exclusion. When we repackage for the client/server model, or dynamically load all our connectors, then we can revisit.